src/V3Gate.cpp

```diff
@@ -860,6 +908,7 @@ class GateInline final {
         : m_graph{graph} {
         // Find gate interconnect and optimize
         graph.userClearVertices();  // vertex->user(): bool. Indicates we've set it as consumed
+        GateWideExprExclude::apply(graph);
```

Comment:
I don't think there is a need to iterate the graph one more time just to check this condition. You can just add a predicate in GateInline and exclude in `optimizeSignals` e.g. right after this line:

https://github.com/verilator/verilator/blob/0c820c30689965aec1b226a41381af9964c42fb6/src/V3Gate.cpp#L767

```
if (excludedWide(vVtxp) {
   ++m_statExcluded;
   continue;
}
```